proposals/0253-callable.md

```diff
@@ -3,8 +3,9 @@
 * Proposal: [SE-0253](https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md)
 * Authors: [Richard Wei](https://github.com/rxwei), [Dan Zheng](https://github.com/dan-zheng)
 * Review Manager: [Chris Lattner](https://github.com/lattner)
-* Status: **Returned for revision**
-* Implementation: [apple/swift#23517](https://github.com/apple/swift/pull/23517)
+* Status: **Awaiting review**
+* Implementation: [apple/swift#24299](https://github.com/apple/swift/pull/24299)
+* Previous Revisions: [[1](https://github.com/apple/swift-evolution/blob/36ea8be09508db9380949954d0c7a101fdb15226/proposals/0253-callable.md)]
 * Decision Notes: [Rationale](https://forums.swift.org/t/returned-for-revision-se-0253-static-callables/23290)
 
 ## Introduction

```

Comment:
Similarly in the intro, Swift already has callable values.

Reply:
" the keyword call:" -> not anymore

Reply:
Done.